### PR TITLE
added support for Dyson Pure Cool.

### PR DESCRIPTION
**Pull request by hvtuananh**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -107,6 +107,21 @@ def update(self, name: str, serial: str, message: object) -> None:
       self.heat_target.labels(name=name, serial=serial).set(heat_target)
       self.quality_target.labels(name=name, serial=serial).set(message.quality_target)
       self.filter_life.labels(name=name, serial=serial).set(filter_life)
+    elif isinstance(message, dyson_pure_state.DysonPureCoolState):
```
Comment (seanrees) [inline main.py]: What do you think about combining the elif's on line 88 and 110?

DysonPureHotCoolState inherits from DysonPureCoolState, so something like this should work and reduce code duplication:

```
elif isinstance(message, dyson_pure_state.DysonPureCoolState):
  self.fan_mode.labels(...)
  (rest of the common PureCool code)

  if isinstance(message, dyson_pure_state.DysonPureHotCoolState):
    self.heat_mode.labels(...)
    (rest of the heat code)
```

**Review by seanrees — CHANGES_REQUESTED** [review]
Thanks again! Just tossed an idea in there to see if we can reduce some code duplication :-)

File: main.py (hunk 1 above)
Comment (hvtuananh) [reply main.py]: Yes, thank you for the suggestion! I've updated the PR. Looks much better now.

**Comment by seanrees** [comment]
Merged, thanks!

**Pull request merged**